src/main/java/org/fundacionjala/coding/joseColina/EANValidator.java

```diff
@@ -0,0 +1,42 @@
+package org.fundacionjala.coding.joseColina;
+
+public final class EANValidator {
+    private static final int RADIX_TEN = 10;
+    private static final int ODD_MULTIPLIER = 1;
+    private static final int EVEN_MULTIPLIER = 3;
+
+    private EANValidator() {
+    }
+    public static boolean validate(final String eavnNumber) {
```

Comment:
You can fix the problem using a single for loop.
so the getDigits method will no longer needed